apps/nerves_hub_www/lib/nerves_hub_www_web/live/device_live/index.ex

```diff
@@ -100,6 +100,21 @@ defmodule NervesHubWWWWeb.DeviceLive.Index do
     {:noreply, socket}
   end
 
+  def handle_event(
+        "reboot",
+        %{"device-id" => device_id},
+        %{assigns: %{devices: devices, user: user}} = socket
+      ) do
+    user = Repo.preload(user, :org_users)
+
+    device = Enum.find(devices, fn device -> device.id == String.to_integer(device_id) end)
```

Comment:
This isn't a big deal but we can pull the `String.to_integer` function out of the enumerator so that it is only executed once instead of once every loop.

```elixir
device_id = String.to_integer(device_id)
device = Enum.find(devices, fn device -> device.id == device_id end)
```